libs/utils/decorate.js

```diff
@@ -228,12 +228,73 @@ function getVideoAttrs(hash, dataset) {
   return `${globalAttrs} controls`;
 }
 
+export function syncPausePlayIcon(video) {
+  const playIcon = video.nextElementSibling?.querySelector('.play-icon');
+  const pauseIcon = video.nextElementSibling?.querySelector('.pause-icon');
+  if (video.paused || video.ended) {
+    playIcon?.classList.remove('hidden');
+    pauseIcon?.classList.add('hidden');
+  } else {
+    pauseIcon?.classList.remove('hidden');
+    playIcon?.classList.add('hidden');
+  }
+}
+
+export function addAccessibilityControl(videoString, videoAttributes, tabIndex = 0) {
+  if (!videoAttributes.includes('controls')) {
+    if (videoAttributes.includes('hoverplay')) {
+      return `<a class='pause-play-wrapper' tabindex=${tabIndex} alt='play/pause motion' aria-label='play/pause motion' >${videoString}
```

Comment:
Would an `alt` attribute achieve anything here? It's likely that the `aria-label` one is enough. We might also want to adapt this to use a placeholder so we cover all languages.